scripts/create_gitlab_snapshot.py

```diff
@@ -38,24 +39,50 @@
 def main(argv):
     import argparse
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument('--no-restart',
-                        dest='restart',
-                        default=True,
-                        action='store_false',
-                        help='Leave the EC2 instance in a stopped state after '
-                             'the snapshot has been created.')
+    g = parser.add_mutually_exclusive_group()
+    g.add_argument('--no-restart',
+                   dest='restart',
+                   default=True,
+                   action='store_false',
+                   help='Leave the EC2 instance in stopped state after '
+                        'the snapshot has been created.')
+    g.add_argument('--start-only',
+                   dest='start',
+                   default=False,
+                   action='store_true',
+                   help='Start the EC2 instance from the stopped state. '
+                        'Fails if the instance is not stopped.')
+    g.add_argument('--new-size',
+                   dest='size',
+                   action='store',
+                   type=int,
+                   help='After creating a new snapshot, restore the snapshot to a new'
+                        'data volume of the given size (in GiB).')
```

Comment:
Add "The new volume will be tagged so that subsequently deploying the `gitlab` component will automatically discover and attach it to the instance. The old volume must be deleted manually. A shell command to do so will be printed for your convenience."